tests/clubs/test_views.py

```diff
@@ -550,6 +550,24 @@ def test_club_create_no_input(self):
         resp = self.client.post(reverse('clubs-list'), {}, content_type='application/json')
         self.assertIn(resp.status_code, [400, 403], resp.content)
 
+    def test_club_create_nonexistent_tag(self):
+        """
+        Creating a club with nonexistent tags should throw an error.
+        """
+        self.client.login(username=self.user5.username, password='test')
+
+        resp = self.client.post(reverse('clubs-list'), {
+            'name': 'Penn Labs',
+            'description': 'We code stuff.',
+            'email': 'contact@pennlabs.org',
+            'tags': [
+                {
+                    'name': 'totally definitely nonexistent tag'
+                }
+            ]
+        }, content_type='application/json')
+        self.assertIn(resp.status_code, [400, 404], resp.content)
```

Comment:
If the test deterministically results in the same error code, I feel like this should be `assertEquals` here.

Reply:
I'm just future proofing the test case, I feel like either a `400 Bad Request` or a `404 Not Found` would both be valid error messages here. A future change might have us returning a different exception, which could result in the 404 error code. Let me know what you think!